doc/releases/changelog-dev.md

```diff
@@ -72,6 +72,9 @@
 * Removes a patch in `interfaces/autograd.py` that checks for the `strawberryfields.gbs` device.  That device
   is pinned to PennyLane <= v0.29.0, so that patch is no longer necessary.
 
+* Fixed a bug where `qml.map_wires` would not copy the `shots` attribute to a newly-created `QuantumScript`.
```

Comment:
I'm not sure if adding this to the bug-fix section is necessary since shots were only added to the tape after the 0.30 release. You can probably just include the link under the same entry which mentions adding shots to the tape.